packages/execution-environments/jest.config.js

```diff
@@ -2,14 +2,23 @@ module.exports = {
   collectCoverage: true,
   // Ensures that we collect coverage from all source files, not just tested
   // ones.
-  collectCoverageFrom: ['./src/**/*.ts', '!./**/__GENERATED__/**'],
+  // 1. TODO: add coverage for lockdownMore once more research has been done into SES
+  // 2. skipping coverage for iframe and web-worker executors
+  // as the BaseSnapExecutor class is fully tested
+  collectCoverageFrom: [
+    './src/**/*.ts',
+    '!./**/__GENERATED__/**',
+    '!./src/common/lockdown/lockdown-more.ts',
+    '!./src/iframe/**',
```

Comment:
I'm not sure we want to ignore all of these 🤔

Reply:
Not sure if its the correct practice to ignore from the ones you don't intend to collect coverage on but I can remove. Skipped `lockdown-more.ts` since Erik told me to drop it -- have to come back to it once I do some more research. Iframe and WebWorker executors were skipped because I had covered the base class and because WindowsPostMessageStream doesn't seem to work in JSDom. Open to suggestions 🙂